package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
```

Comment:
I really thought CRA fixed this issue already 🤦🏾‍♂️ good job finding  the fix over the weekend.